WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -140,30 +140,27 @@ protected function setup_groups( $key ) {
 	 * @return void
 	 */
 	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
-		$tokens = $phpcsFile->getTokens();
-		$token  = $tokens[ $stackPtr ];
+		$tokens        = $phpcsFile->getTokens();
+		$token         = $tokens[ $stackPtr ];
+		$token_content = strtolower( $token['content'] );
```

Comment:
Is this `$token_content` not unrelated to the rest of the changes going on here? i.e. does it deserve it's own issue (to describe whatever bug this `strtolower()` is fixing) and PR?